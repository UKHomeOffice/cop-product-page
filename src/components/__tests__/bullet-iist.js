import React from "react";
import renderer from "react-test-renderer";
import BulletList from "../bullet-list";

describe("button", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<BulletList />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("Returns list length of 0 when bullet list is empty", () => {
    const tree = renderer.create(<BulletList list={[]} />);
    const testInstance = tree.root;
    expect(testInstance.props.list.length).toBe(0);
  });

  it("Returns list length of 2 when bullet list has 2 list items", () => {
    const tree = renderer.create(<BulletList list={["item 1", "item 2"]} />);
    const testInstance = tree.root;
    expect(testInstance.props.list.length).toBe(2);
  });

  it("Returns list item 2 when second list item is called", () => {
    const tree = renderer.create(<BulletList list={["item 1", "item 2"]} />);
    const testInstance = tree.root;
    expect(testInstance.props.list[1]).toBe("item 2");
  });
});
