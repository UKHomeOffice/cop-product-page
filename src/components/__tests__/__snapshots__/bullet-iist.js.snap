// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button renders correctly 1`] = `
<ul
  className="govuk-list govuk-list--bullet"
/>
`;
